go/flowctl/cmd-discover.go

```diff
@@ -242,14 +243,20 @@ func writeConfigStub(image string, w io.WriteCloser) error {
 
 	var config interface{}
 
+	// Visit leaf-most schema locations first.
+	// Because we're creating yaml.Nodes instead of []interface{}
```

Comment:
Only because I'm curious, but why is that? Seems like a weird restriction

Reply:
It's because Create() knows how to create children of []interface{} or map[string]interface{}, but doesn't know anything about yaml.Nodes or how to create sub-entities therein.